src/components/InputTodo.js

```diff
@@ -0,0 +1,45 @@
+// eslint-disable-next-line max-len
+/* eslint-disable no-alert, react/destructuring-assignment, react/prop-types, react/state-in-constructor */
```

Comment:
Please avoid disabling linters, unless stated otherwise, which is not the case with this project. its  included on the project to help you adhere to coding best practices 👏